can-bus/json/DCM/DCM_rx.json

```
@@ -9,6 +9,7 @@
         "INVR_InternalStates",
         "INVL_MotorPositionInfo",
         "INVR_MotorPositionInfo",
-        "BMS_AvailablePower"
+        "BMS_AvailablePower",
+        "Debug_CanModes"
     ]
 }
```

nit: missing blank line
```suggestion
}

```